src/app/workspace/resource/representation/upload/upload.component.ts

```diff
@@ -283,6 +295,9 @@ export class UploadComponent implements OnInit {
             case 'document':
                 this.allowedFileTypes = this.supportedDocumentTypes;
                 break;
+            case 'audio':
+                this.allowedFileTypes = this.supportedAudioTypes;
+                break;
             default:
```

Comment:
Not severe: In this file there are incongruent blank lines within switch blocks. Either use a blank line after every case block or no blank line after case block

Reply:
resolved in 8f6c0e4